_data/sites.json

```
@@ -6765,7 +6765,7 @@
         "name": "Shutterstock",
         "url": "http://submit.shutterstock.com/privacy.mhtml",
         "difficulty": "hard",
-        "notes": "If you wish to delete your account or request that we no longer use your information to provide you services contact us at Support@Shutterstock.com.",
+        "notes": "If you wish to delete your account or request that we no longer use your information to provide you services contact us at Privacy@shutterstock.com.",
```

```suggestion
        "notes": "If you wish to delete your account or request that they no longer use your information to provide you services, send them an e-mail.",
        "email": "privacy@shutterstock.com",
```
If we use the email field, a button will appear for people to click and send it.

---

Hello @OfficialMuffin, can you please update this so we can merge? Thanks!

---

Sorry, I'll get this done tonight